avalanche.yaml

```diff
@@ -1,7 +1,38 @@
-componentCount: 50
```

Comment:
Is there still value in keeping the old way of configuring things (at least for metrics)? 

The current scale avalanche test script uses the old way, i.e., we can simply specify number of components, metric count, and metric prefix.

I'm fine either way. i.e., we can update the script how it configures and creates the table accordingly.

Reply:
Lets update the script; rather than trying to maintain backward compatibility?

Reply:
My thought was to have a default entity for a component. If 'numComponents` is specified then user has the option to specify entities in each component. If not then component will have a single default component. None of this is coded though. But adding that wont be much work.